thrift-confignode/src/main/thrift/confignode.thrift

```diff
@@ -21,7 +21,20 @@ include "rpc.thrift"
 namespace java org.apache.iotdb.confignode.rpc.thrift
 namespace py iotdb.thrift.confignode
 
-struct SetStorageGroupoReq {
+struct EndPoint {
+  1: required string ip
+  2: required i32 port
+}
+
+// The return status code and message in each response.
+struct TSStatus {
```

Comment:
The TSStatus is already defined in org.apache.iotdb.service.rpc.thrift. It can be just included. If there is a necessity to define a different Status code for ConfigNode, it should be renamed.

Reply:
if delete struct TSStatus, compile will be failed.   include "rpc.thrift"  does not take effect.

Reply:
Then can we rename this TSStatus as ConfigNodeStatus in order to distinguish between the TSStatus in rpc.thrift?